Sources/SKSupport/LineTable.swift

```diff
@@ -141,6 +141,20 @@ extension LineTable {
     return content.utf16.index(lineSlice.startIndex, offsetBy: utf16Column, limitedBy: lineSlice.endIndex)
   }
 
+  /// Returns `String.Index` of given logical position.
+  ///
+  /// - parameter line: Line number (zero-based).
+  /// - parameter utf8Column: UTF-8 column offset (zero-based).
```

Comment:
Zero based line/column is super weird. Do we actually need it to be?

Reply:
All of LSP is zero-based, so we tend to default to that as well.